mlkem/ntt.c

```diff
@@ -95,37 +117,54 @@ void poly_ntt(poly *p) { ntt_native(p); }
  * Description: Computes inverse of negacyclic number-theoretic transform (NTT)
  *              of a polynomial in place;
  *              inputs assumed to be in bitreversed order, output in normal
- *order
+ *              order
+ *
+ *              The input is assumed to be in bitreversed order.
+ *
+ *              The output polynomail is in normal order, and
+ *              coefficient-wise bound by INVNTT_BOUND in absolute value.
+ *
+ *              Values of INVNTT_BOUND:
+ *              - C: 3/4 q
```

Comment:
As above - I think the abstraction should be the same for all backends.

Reply:
Done